### PR TITLE
update to Xcode 12

### PR DESCRIPTION
**Pull request by paulz**

### CODE REVIEW

File: .github/workflows/deflake-ui-tests-on-self-hosted.yml [hunk 1]
```diff
@@ -18,11 +18,11 @@ defaults:
 jobs:
   build-then-test:
 
-    runs-on: [self-hosted, xcode11]
+    runs-on: macos-latest
```
Comment (paulz) [inline .github/workflows/deflake-ui-tests-on-self-hosted.yml:21]: this workflow have to run on self hosted if we change label to `macos-latest` then it goes to github 

please use `[self-hosted, x86_64]`

File: .github/workflows/self-hosted.yml [hunk 2]
```diff
@@ -10,11 +10,11 @@ defaults:
 jobs:
   build-then-test:
 
-    runs-on: [self-hosted, xcode11]
+    runs-on: macos-latest
```
Comment (paulz) [inline .github/workflows/self-hosted.yml:13]: same here, use self hosted with newer labels

File: UITests/UpdateAppTest.swift [hunk 3]
```diff
@@ -76,7 +76,7 @@ class UpdateAppTest: XCTestCase {
         ])
     }
 
-    func testAutoUpgrade() throws {
+    func _testAutoUpgrade() throws {
```
Comment (paulz) [inline UITests/UpdateAppTest.swift:79]: that should pass on self hosted, please re-enable

File: .github/workflows/deflake-ui-tests-on-self-hosted.yml (hunk 1 above)
Comment (ruslan-qualtie) [reply .github/workflows/deflake-ui-tests-on-self-hosted.yml]: Unfortunately it didn't help.
`No runner matching the specified labels was found: self-hosted, x86_64.`
https://github.com/sparta-science/connect/actions/runs/725209184

**Pull request merged**